pkg/operator/yugabytedb/controller.go

```diff
@@ -182,8 +182,13 @@ func validateClusterSpec(spec yugabytedbv1alpha1.YBClusterSpec) error {
 		return fmt.Errorf("VolumeClaimTemplate unavailable in TServer spec.")
 	}
 
-	validateResourceSpec(spec.Master.Resource, false)
-	validateResourceSpec(spec.TServer.Resource, true)
+	if err := validateResourceSpec(spec.Master.Resource, false); err != nil {
+		return fmt.Errorf("failed to validate resource spec %v", err)
```

Comment:
`errors.Wrap()` is preferred instead of `fmt.Errorf()` with the formatting of the error.
```suggestion
		return errors.Wrap(err, "failed to validate resource spec %v")
```

Reply:
yes, I'll squash in single commit